packages/babel-parser/src/parser/error-message.js

```diff
@@ -16,8 +16,9 @@ export const ErrorMessages = Object.freeze({
     "Can not use 'await' as identifier inside an async function",
   AwaitExpressionFormalParameter:
     "await is not allowed in async function parameters",
-  AwaitNotInAsyncFunction:
-    "Can not use keyword 'await' outside an async function",
+  AwaitNotInAsyncContext:
+    "'await' is only valid in async functions and the top level bodies of modules",
```

Comment:
Small nit, but thoughts on:

```suggestion
    "'await' is only allowed within async functions and at the top levels of modules",
```